src/components/GamePage.tsx

```
@@ -87,22 +118,19 @@ const Game: React.FC = () => {
       }
-      
-      // console.log("winner is: ");
-      // console.log(winner);
 
-      if(retries==50 && winner=='')
-      setError('DeepSeek Failed to make a move. Please restart the game.');
-    } catch {
-      setError('An error occurred during AI’s turn.');
+      if (retries === 50 && winner === '')
+        showAlert('DeepSeek Failed to make a move. Please restart the game.', 'error');
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)